sqlserver/datadog_checks/sqlserver/utils.py

```diff
@@ -25,6 +25,13 @@ def construct_use_statement(database):
     return 'use [{}]'.format(database)
 
 
+def get_stmt_is_proc(text):
+    if text:
+        t = text.upper()
```

Comment:
I do think taking the substring of the first 500 characters will help reduce false positives on very long procedures.

Reply:
ah yeah, good call

Reply:
>  I do think taking the substring of the first 500 characters will help reduce false positives on very long procedures.

why the first 500 chars? possible that creates weird edge cases where someone has a really long comment at the beginning of the sql text?